CoreFoundation/Base.subproj/CFKnownLocations.c

```diff
@@ -0,0 +1,75 @@
+/*	CFKnownLocations.c
+	Copyright (c) 1999-2017, Apple Inc. and the Swift project authors
+ 
+	Portions Copyright (c) 2014-2017, Apple Inc. and the Swift project authors
+	Licensed under Apache License v2.0 with Runtime Library Exception
+	See http://swift.org/LICENSE.txt for license information
+	See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+*/
+
+#include "CFKnownLocations.h"
+
+#include <CoreFoundation/CFString.h>
+#include "CFPriv.h"
+#include "CFInternal.h"
+
+#include <assert.h>
+
+CONST_STRING_DECL(_kCFKnownLocationUserAny, " == _kCFKnownLocationUserAny");
```

Comment:
Hm, the ` == ` at the start is something unusual. In most cases the string is equal to the key name.

Reply:
Yeah — the idea here was to make sure it couldn't be a valid username. Since we're switching to an enum, this problem goes away.